server/connectors/api-connector/src/main/scala/com/prisma/api/connector/DataResolverTypes.scala

```diff
@@ -115,5 +115,6 @@ object AtLeastOneRelatedNode extends RelationCondition
 object NoRelatedNode         extends RelationCondition
 object NoRelationCondition   extends RelationCondition
 
-case class NodeSubscriptionFilter()                        extends Filter
-case class PreComputedSubscriptionFilter(boolean: Boolean) extends Filter
+case class NodeSubscriptionFilter() extends Filter
+case class TrueFilter()             extends Filter
+case class FalseFilter()            extends Filter
```

Comment:
Those 3 filters can all be replaced with singleton objects as they do not have any parameters.